addons/event/views/event_views.xml

```diff
@@ -533,7 +533,6 @@
                         <div class="oe_button_box" name="button_box"/>
                         <group>
                             <group string="Attendee" name="attendee">
-                                <field name="partner_id" attrs="{'readonly':[('state', '!=', 'draft')]}"/>
```

Comment:
Actually field was duplicated ... could be great to explain it in commit message (I thought you were removing it ^^ ) . By the way, don't we keep the attrs ?

Reply:
It is already mentioned in the commit that the field has been changed its position next to the ticket and we don't need to keep the attrs because in the specification it clearly mentioned to keep this field editable.

Reply:
I think there is indeed a mismatch in the commits, the field "partner_id" is added (and duplicated) in an earlier commit, then removed in this commit.